cmd/fyne_demo/screens/container.go

```diff
@@ -143,3 +148,34 @@ func makeSplitTab() fyne.CanvasObject {
 	)
 	return container.NewHSplit(container.NewVScroll(left), right)
 }
+
+func makeTreeTab() fyne.CanvasObject {
+	left := make(map[string][]string)
+	internalwidget.AddTreePath(left, "A", "B", "C", "abc")
+	internalwidget.AddTreePath(left, "A", "D", "E", "F", "adef")
+	internalwidget.AddTreePath(left, "A", "D", "E", "G", "adeg")
+	internalwidget.AddTreePath(left, "A", "H", "I", "ahi")
+	internalwidget.AddTreePath(left, "A", "J", "K", "ajk")
+	internalwidget.AddTreePath(left, "A", "L", "M", "N", "almn")
+	internalwidget.AddTreePath(left, "A", "O", "ao")
+	internalwidget.AddTreePath(left, "A", "P", "Q", "R", "apqr")
+	internalwidget.AddTreePath(left, "A", "S", "T", "U", "astu")
+	internalwidget.AddTreePath(left, "A", "V", "W", "X", "Y", "Z", "avwxyz")
+
+	l := widget.NewTreeWithStrings(left)
+	l.OnNodeSelected = func(id string, node fyne.CanvasObject) {
+		fmt.Println("LeftTreeNodeSelected:", id)
+	}
+	l.OpenBranch("A")
+	l.OpenBranch("D")
+	l.OpenBranch("E")
+	l.OpenBranch("L")
+	l.OpenBranch("M")
+
+	dir := filepath.Join(os.Getenv("GOPATH"), "src", "fyne.io", "fyne")
```

Comment:
Probably not a good path for a binary that could be run on any computer with or without Go or Fyne installed..